chain/indexer/src/streamer/utils.rs

```diff
@@ -14,7 +14,7 @@ pub(crate) async fn convert_transactions_sir_into_local_receipts(
     runtime_config: &RuntimeConfig,
     txs: Vec<&IndexerTransactionWithOutcome>,
     block: &views::BlockView,
-    protocol_version: ProtocolVersion,
+    _protocol_version: ProtocolVersion,
```

Comment:
Remove unused parameter here and at all callsites of `convert_transactions_sir_into_local_receipts`